config/application.rb

```diff
@@ -21,6 +21,9 @@ class Application < Rails::Application
 
     # don't generate RSpec tests for views and helpers
     config.generators do |g|
+      g.template_engine :erb
```

Comment:
we're actually using haml not sure why this is necessary

Reply:
Yup. Forgot to take this out. I like to write in `erb` and convert to `haml` since `erb` is faster for me.